File: December 09/dec9.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int words = 0;
    string str;
    cout<<"enter String:";
    cin>>str;
    
    for(int i=0; i<str.size(); i++) 
    {
        if(isupper(str[i]))
        {
            words++;
        }
            
    }
    
    cout<<words;
    
    return 0;
}

File: December 09/dec9.java
import java.util.Scanner;

public class dec9 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the camel case string: ");
        String camelCaseString = scanner.next();

        int wordCount = 1; 
        for (int i = 0; i < camelCaseString.length(); i++) {
            char currentChar = camelCaseString.charAt(i);

          
            if (Character.isUpperCase(currentChar)) {
                wordCount++;
            }
        }
        System.out.println("Number of words: " + wordCount);

        scanner.close();
    }
}